glib/src/subclass/object.rs

```diff
@@ -133,6 +133,52 @@ unsafe extern "C" fn dispose<T: ObjectImpl>(obj: *mut gobject_ffi::GObject) {
     }
 }
 
+// rustdoc-stripper-ignore-next
+/// Trait containing only the property related functions of `ObjectImpl`.
+/// Implemented by the `Props` macro.
+/// When implementing `ObjectImpl` you may want to delegate the function calls to this trait.
+/// Some functions have a different return type from `ObjectImpl` to handle possible errors caused by missing
+/// properties.
+pub trait AutoObjectProps: ObjectSubclass {
```

Comment:
```suggestion
pub trait AutoObjectProperties: ObjectSubclass {
```

and elsewhere? We don't have to pay taxes per character :)

Reply:
Also I wouldn't prefix the traits with Auto.

Reply:
`Auto` makes it clear the trait shouldn't be implemented by the user